src/components/InactivityTracker/InactivityTracker.js

```diff
@@ -14,12 +13,14 @@ export const InactivityTracker = ({ openedFacilityTabs, setCheckout }) => {
   const [showInactiveModal, setShowInactiveModal] = useState(false);
 
   const isFacilityCheckedOut = () => {
-    return (
-      openedFacilityTabs.length > 0 && openedFacilityTabs[0].checkout === true
-    );
+    for (let i = 0; i < openedFacilityTabs.length; i++) {
```

Comment:
this function can be simplified down to 1 line:

const isFacilityCheckedOut = () => {
  return (openedFacilityTabs.find((element) => element.checkout === true).length > 0);
};